experimental/QuadFormAndIsom/src/embeddings.jl

```diff
@@ -871,19 +1010,19 @@ to $M$, and $N'$ is the orthogonal complement of $M'$ in $L'$.
 If `T == false`, then $V$ will always be the empty list. If `T == true`, then
 the content of $V$ depends on the value of the symbol `classification`. There
 are 4 possibilities:
-  - `classification == :none`: $V$ is the empty list;
-  - `classification == :first`: $V$ consists of the first primitive embedding found;
-  - `classification == :sublat`: $V$ consists of representatives for all isomorphism classes of primitive embeddings of $M$ in $L$, up to the actions of $O(M)$ and $O(q)$ where $q$ is the discriminant group of $L$;
-  - `classification == :emb`: $V$ consists of representatives for all isomorphism classes of primitive embeddings of $M$ in $L$ up to the action of $O(q)$ where $q$ is the discriminant group of $L$.
+  - `classification == "none"`: $V$ is the empty list;
+  - `classification == "first"`: $V$ consists of the first primitive embedding found;
+  - `classification == "sub"`: $V$ consists of representatives for all isomorphism classes of primitive embeddings of $M$ in $L$, up to the actions of $O(M)$ and $O(q)$ where $q$ is the discriminant group of $L$;
+  - `classification == "emb"`: $V$ consists of representatives for all isomorphism classes of primitive embeddings of $M$ in $L$ up to the action of $O(q)$ where $q$ is the discriminant group of $L$.
 
 If `check` is set to `true`, the function determines whether $L$ is in fact unique
 in its genus.
 
 We follow the algorithm described in the proof of Proposition 1.15.1 of
-[Nik79](@cite). The classification methods for the symbols `:sublat` and `:emb`
+[Nik79](@cite). The classification methods for the symbols `"sub"` and `"emb"`
```

Comment:
Do we have a guideline whether to use strings or symbols for keyword arguments?

Reply:
Most other parts of the code I know of use symbols